_targets.R

```diff
@@ -0,0 +1,44 @@
+# pipeline script
+
+#load library
+library(targets)
+
+#source functions
+source('1_fetch/src/01_get_data.R')
```

Comment:
No necessary change here, but just a comment - since each of these R script files are nested _within_ a numbered phase, it isn't necessary to include the number in their name.  E.g. `'1_fetch/src/01_get_data.R'` could just be `'1_fetch/src/get_data.R'`